packages/client/hmi-client/src/components/workflow/tera-dataset-node.vue

```diff
@@ -84,4 +80,17 @@ span {
 	font-size: var(--font-caption);
 	color: var(--text-color-subdued);
 }
+
+.p-button-sm.p-button-outlined {
+	border: 1px solid var(--surface-border);
```

Comment:
This is done in `tera-model-node.vue` as well. Do you think it'd be worth applying these rules into prime vue SASS instead?